test/integrations/destinations/mp/router/data.ts

```diff
@@ -0,0 +1,898 @@
+const defaultMockFns = () => {
+  jest.spyOn(Date, 'now').mockImplementation(() => new Date(Date.UTC(2020, 0, 25)).valueOf());
+};
+
+const sampleDestination = {
```

Comment:
possible to create this in common file for processor and router.

Reply:
Created and moved it into that.